printer_mods/arhodiewithsoul/Bed/readme.md

```diff
@@ -0,0 +1,9 @@
+Contains updated Leadscrew Block and motor mounts for non-spec 185x185 (Prusa Mini), 200x200 (XYZPrinting DaVinci) build plates
+
```

Comment:
There are no leadscrew block STLs in this directory, just the motor mounts.

Why are the motor mounts and leadscrew blocks different on a smaller bed?  Would the stock parts work?

Z motor mount plates have holes that would print in air on the bottom, the need more chamfer:
<img width="370" alt="Screen Shot 2021-05-02 at 17 18 46" src="https://user-images.githubusercontent.com/69389593/116828096-920ada80-ab6a-11eb-81c8-8543433d0439.png">

Reply:
Updated the readme - the standard leadscrew mounts interfere with the shaft retainers either side. These mounts are narrower and therefore allow for clearance to their neighbouring parts.